bgp-monitor-action-pack/config.yaml

```diff
@@ -1,4 +1,4 @@
 type: PACKAGE
-version: 1.3.0
+version: 1.3.2
```

Comment:
This needs to be 1.4.0 as it's an update to the functionality rather than a bug fix

Reply:
I updated to 1.4.0 and pushed.